app/views/emails/index.js

```diff
@@ -0,0 +1 @@
+$('#emails').html('<%= escape_javascript(options_for_select(@emails)) %>');
```

Comment:
GET requests to `/emails` will respond with the JavaScript generated by this template. This JavaScript replaces the html content of `<datalist id="emails">` with `<option>` elements containing email values.